lisp/mastodon-toot.el

```diff
@@ -142,14 +141,53 @@ Set `mastodon-toot--content-warning' to nil."
       (mastodon-http--triage response
                              (lambda () (message "Toot toot!"))))))
 
+(defun mastodon-toot--get-acct()
```

Comment:
Unused function.

Reply:
This function is meant to be used interactively by the user. Is there a specific way to mark such a function?